MTM-59931 Support for Additional Special Characters in MQTT Topics (#2324)
SoftwareAG/c8y-docs: content/device-integration/mqtt-service-bundle/implementation.md

### Topic {#topic}

> MQTT Service topics are mapped to the Messaging Service subscriptions with identical names, including additional URL encoding.

Reviewer: **_nitpick:_**

```suggestion
MQTT service topics are mapped to Messaging Service subscriptions with matching names, including any additional URL encoding.
```

Other than that you are free to use any topic name which is compatible with the [MQTT specification](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718106).

{{< c8y-admon-info >}}

> Wildcard topics (`+`, `#`) and system topics starting with `$` are not supported.

Reviewer: **_nitpick:_**

```suggestion
Wildcard topics (`+`, `#`) and system topics that begin with `$` are not supported.
```
